### PR TITLE
Forcepoint dlp

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6113

### CODE REVIEW

File: Packs/ForcepointDLP/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# <Product Name>
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+Use the Settings > General > Remediation page in the Data Security module of the Forcepoint Security Manager to define the location of the syslog server and mail release gateway used for remediation.
+
+1. Under Syslog Settings, enter the IP address or hostname of the syslog server, and the logging Port.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:8]:
```suggestion
1. Under Syslog Settings, enter the IP address or hostname of the syslog server, and the logging port. 
```

File: Packs/ForcepointDLP/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+Use the Settings > General > Remediation page in the Data Security module of the Forcepoint Security Manager to define the location of the syslog server and mail release gateway used for remediation.
+
+1. Under Syslog Settings, enter the IP address or hostname of the syslog server, and the logging Port. 
+2. To set the origin of syslog messages, select Use syslog facility for these messages, then use the drop-down menu to select the type of message to appear in the syslog:
+   * User-level Messages (#1) logs generic user-level messages, such as "username/password expired".
+   * Security/Authorization Messages (#4) logs authentication- and authorization-related commands, such as "authentication failed for admin user".
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:11]:
```suggestion
   * **Security/Authorization Messages (#4)** logs authentication and authorization-related commands, such as "authentication failed for admin user".
```

File: Packs/ForcepointDLP/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# <Product Name>
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+Use the Settings > General > Remediation page in the Data Security module of the Forcepoint Security Manager to define the location of the syslog server and mail release gateway used for remediation.
+
+1. Under Syslog Settings, enter the IP address or hostname of the syslog server, and the logging Port. 
+2. To set the origin of syslog messages, select Use syslog facility for these messages, then use the drop-down menu to select the type of message to appear in the syslog:
+   * User-level Messages (#1) logs generic user-level messages, such as "username/password expired".
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:10]:
```suggestion
   * **User-level Messages (#1)** logs generic user-level messages, such as "username/password expired".
```

File: Packs/ForcepointDLP/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+   * Security/Authorization Messages (#4) logs authentication- and authorization-related commands, such as "authentication failed for admin user".
+   * Security/Authorization Messages (#10) logs non-system authorization messages inside a protected file (for information of a sensitive nature, such as passwords).
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:12]:
```suggestion
   * **Security/Authorization Messages (#10)** logs non-system authorization messages inside a protected file (for information of a sensitive nature, such as passwords).
```

File: Packs/ForcepointDLP/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+   * Local use 0-7 (#16-23) specifies unreserved facilities available for any local use. Processes and daemons that have not been explicitly assigned a facility can use any of the "local use" facilities. Configuration is done in the syslog.conf file.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:13]:
```suggestion
   * **Local use 0-7 (#16-23)** specifies unreserved facilities available for any local use. Processes and daemons that have not been explicitly assigned a facility can use any of the "local use" facilities. Configuration is done in the syslog.conf file.
```

File: Packs/ForcepointDLP/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+   To send incident data to the syslog, select Audit Incident > Send Syslog Message in the action plan for the policy.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:14]:
```suggestion
   To send incident data to the syslog, select **Audit Incident** > **Send Syslog Message** in the action plan for the policy.
```

File: Packs/ForcepointDLP/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+3. Click Test Connection to send the syslog server a verification test message.
+4. Under Release Quarantined Emails, specify which gateway to use when releasing a quarantined email message.
+   * The default is Use the gateway that detected the incident. This gateway could be Forcepoint Email Security or the protector MTA, depending on your subscription.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:17]:
```suggestion
   * The default is **Use the gateway that detected the incident**. This gateway could be Forcepoint Email Security or the protector MTA, depending on your subscription.
```

File: Packs/ForcepointDLP/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+   * To define a specific gateway, select Use the following gateway, then enter the gateway IP address or hostname and Port.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:18]:
```suggestion
   * To define a specific gateway, select **Use the following gateway**, then enter the gateway IP address or hostname and port.
```

File: Packs/ForcepointDLP/README.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+5. If only recipients of a message should be able to release it from quarantine, select Validate user before releasing message.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:19]:
```suggestion
5. If only recipients of a message should be able to release it from quarantine, select **Validate user before releasing message**.
```

File: Packs/ForcepointDLP/README.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+The system then ensures that the person attempting to release a message is a recipient of the message, and therefore authorized. Unauthorized users receive an email notification that they are not allowed release the message.
+6. Click OK to save your changes.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointDLP/README.md:21]:
```suggestion
6. Click **OK** to save your changes.
```

**Comment by ShirleyDenkberg** [comment]
@yasta5 Doc review completed.

**Review by nkanon — APPROVED** [review]

**Pull request merged**